### ISSUE TITLE
Downgrading 1.14b is not working.

### ISSUE THREAD

**Issue by ambroiseRabier** [ChaosMarc/D2VersionChanger#2]
> Caution: If installed with the new v1.14 digital installers by Blizzard you cannot downgrade your Diablo 2 installation to older patches. When starting the game you will receive only an empty error window. To fix this either use the included MPQFix or install the game with your CDs [...]

In other words, you can install v1.14, but you either need to use the MPQ fix with blizzard patch starter, or start from older patch CD installation.

Using Windows 10.

1. I started with 1.14b patch US version.
2. Extracted D2VersionChanger inside the Diablo II (with LOD installed) directory.
3. Launching the game work (LOD).
4. Launching VersionChanger.bat open the cmd window as expected.
5. I change my version of LOD to 1.13c (and apply). Files get changed or added. (if I launch game now, I get an expected corrupted file)
6. I use the MPQ patch.
7. Launching Game.exe give error:

```
Diablo II Exception

UNHANDLED EXCEPTION:
ACCESS_VIOLATION (c0000005)
```

Inverting MPQ patch with LOD 1.13c version give same result.
Trying to run with XP Service pack 3 compatibily mode give me a box inviting me to run `D2VidTst.exe`.

The source of the mpq patch you provided state:
```
Text Instructions
1. Install Diablo II from Blizzard's Installer [1.12+]
2. Download WinMPQ w/ Fix file and extract the contents of the WinMPQ folder directly into the Diablo II root folder. You can delete these files after we are done.
```

I'll try https://github.com/fearedbliss/Cactus , that is the tool the MPQ patch was made for given the post you linked in readme: https://www.diabloii.net/forums/threads/tech-support-installing-from-bnet-1-14d-installer-doesnt-allow-switching-back-to-1-13d.960439/#post-8827207

Edit: The game run in 1.13c with https://github.com/fearedbliss/Cactus method.

**Comment by ambroiseRabier** [comment]
You might find that helpful to fix the issue: I currently installing plugy for usage of https://github.com/whipowill/d2-plugy-qol, based on a Cactus install. I had the same error at launch, I have modified `PlugY.exe` to run in Windows XP SP3 compatibility mode (run automatically in admin), and modified `PlugY.ini` to have:

```
Param=-3dfx -direct -txt
```

The game stopped with that error and is running when clicking on `PlugY.exe`.

**Comment by ChaosMarc** [comment]
Just wanted to let you know that I will look into this in a few days. Currently I'm swamped with work.

**Comment by ChaosMarc** [comment]
Sorry this took me so long. Your issue should be fixed with v4.5.1
